ksso/server.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import os
+import sys
+
+import flask.cli
+import jwt
+import requests
+from flask import Flask, redirect, request
+
+if getattr(sys, "frozen", False):
+    base_path = sys._MEIPASS  # PyInstaller temp directory
```

Comment:
Can you add a comment that this lines are related to PyInstaller binary?

Reply:
Done